packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -534,6 +534,11 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         );
 
         // Add relay data.
+        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
```

Comment:
By adding the request time to the ancillary data we can compare it against the time passed into `getHistoricalPrice` for further validation. This is useful for two reasons:

- By adding the timestamp to the ancillary data, any slow relayer who had their relay disputed can now submit another relay for the same deposit once the block time has changed.
- (maybe) This provides additional validation that a relay's ancillary data is valid

Reply:
For 1, shouldn't this be the case regardless? The DVM/OO will consider these two distinct requests since their timestamp is different even if we don't include it in ancillary data.